exporter/metric/metric.go

```diff
@@ -126,6 +128,27 @@ func (me *metricExporter) Export(ctx context.Context, rm metricdata.ResourceMetr
 	)
 }
 
+// Temporality returns the Temporality to use for an instrument kind.
+func (me *metricExporter) Temporality(k metric.InstrumentKind) metricdata.Temporality {
```

Comment:
both `Temporality` and `Aggregator` are two new interface methods and this is the bit i need some guidance on how they should work with gcp

Reply:
We should use cumulative temporality and the default aggregators.  Those should be the same as the OTLP exporter's defaults.

Reply:
should the temporality not depend on the instrument kind? felt like Cumulative would fit best gauges and Delta fit counters

Reply:
GCP currently uses cumulative counters.